task/real/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "task-real"
```

Comment:
Is this a test of some kind?

Reply:
Yes, this was an old test. I'll remove it.